releases/release-announcement-guide.md

```diff
@@ -0,0 +1,306 @@
+# Kubeflow’s Release Process Addendum - Release Blog Post and Presentation
+
+
+# Purpose
+
+This document reviews the Kubeflow Community’s process for a blog post and a recorded presentation, which are delivered with each Kubeflow release.   The blog post and presentation are used to increase the Community’s understanding of new features in a Kubeflow release.  
+
+
+## Initial note
+
+Both of these deliveries (blog and presentation) have much of the same content.   The presentation can usually be completed in one shot with some quick edits.  It provides valuable information that can be used in the blog.  The blog requires more precision, and therefore takes multiple iterations.  It is suggested to start the blog before the presentation.
+
+
+# Blog Post
+
+
+## Blog Overview
+
+
+
+* Publish a blog of 1,000 words or less that reviews the highlights of new release
+* The blog will promote themes and the release’s top features.  The blog is a summary of the release, and more detailed information is provided in links to GitHub Issues, Pull Requests (PR), additional (Working Group or User) blogs, repos or other resources.
+* The post uses the GitHub PR fast pages process, which documented here, ​​https://github.com/kubeflow/blog#blog-submission-instructions
+    * This PR is an example of the 1.4 blog post, [https://github.com/kubeflow/blog/pull/109](https://github.com/kubeflow/blog/pull/109) 
+
+
+## Community Resources needed
+
+
+
+* Product Manager - 20 hours
+    * Manages the blog’s creation and release process,  creates initial version, asks and incorporates edits, keeps process moving, provides updates to Community
+* GitHub specialist - 10 hours
+    * Creates PR, helps to check content, incorporate edits to PR
```

Comment:
Could you clarify why a GitHub specialist is needed? Couldn't we combine these two roles since the product manager is the one who writes and reiterates the post based on feedback?

If the concern is that the product manager may not have git skills, I think we should also consider the possibility that GitHub specialist might not be able to edit the PR as effectively as someone who understand the content of the feedback

Reply:
I documented the current process.   If someone has the git and kubeflow content skills then then the GitHub specialists role could be collapsed into the PM.   Note - It helps to have two people working together because last minute changes and updates often cause re-writes.